aea/registries/resources.py

```diff
@@ -65,16 +69,14 @@ def __init__(self, directory: Optional[Union[str, os.PathLike]] = None):
         :param directory: the path to the directory which contains the resources
              (skills, connections and protocols)
         """
-        self._contract_registry = ContractRegistry()
-        self._protocol_registry = ProtocolRegistry()
+        self._component_registry = AgentComponentRegistry()
         self._handler_registry = HandlerRegistry()
         self._behaviour_registry = ComponentRegistry[Behaviour]()
         self._model_registry = ComponentRegistry[Model]()
         self._skills = dict()  # type: Dict[SkillId, Skill]
```

Comment:
Still an odd split? I.e. Skills treated very differently

Reply:
Ok for now though!

Reply:
yes, incoming PR will fix that